pipeline.py

```diff
@@ -1,7 +1,7 @@
 """
 Pipeline for text processing implementation
```

Comment:
You are working with a wrong file
You are supposed to implement scrapper logic during current part of laboratory work